tools/ci_build/github/azure-pipelines/anybuild.yml

```diff
@@ -0,0 +1,150 @@
+# TODO: update to run nightly or something
+trigger: none
+
+parameters:
+  - name: AnyBuildExtraArgs
+    type: string
+    default: --Epoch 0
+
+  - name: OnnxBuildOptions
+    type: string
+    default: --cmake_generator Ninja --skip_submodule_sync --build_shared_lib --config Debug Release
+
+variables:
+  # the location from where the AnyBuild client is downloaded
+  AnyBuildSource: https://anybuild.azureedge.net/clientreleases
+
+  # the AnyBuild cluster url (contains 8 4-core machines)
+  AnyBuildClusterId: https://westus2.anybuild.microsoft.com/clusters/964acefe-3118-4838-8221-380dece043b9
+
+  # set parallelism to 2 * total_number_of_cores_in_the_cluster (= 2 * 8 * 4)
+  AnyBuildDegreeOfParallelism: 64
+
+  # how many cache lookups to run in parallel (the default value below works well for a 2-core machine)
+  AnyBuildMaxParallelLookups: 15
+
+  # just a symbolic name for a local Docker image (this pipeline build this image, then uses it to run a build in a container; it never pushes this image)
+  DockerImageTag: manylinux-onnxr:latest
+
+  # directory where the onnxruntime sources are checked out
+  OnnxSourcesDir: $(Build.SourcesDirectory)/onnxruntime
+
+  # App used for authentication
+  # https://ms.portal.azure.com/#blade/Microsoft_AAD_RegisteredApps/ApplicationMenuBlade/Overview/appId/ef5b3365-5c24-4e2e-b79d-6382a48d7436/isMSAApp/
+  OnnxPrincipalAppId: ef5b3365-5c24-4e2e-b79d-6382a48d7436
+
+  # TODO: create a secret for the App above and save it as a secret pipeline variable in ADO (not in plain-text here)
```

Comment:
This TODO can also go.  It's fine to say that there is a secret variable with this name defined in the corresponding ADO pipeline